include/git2/errors.h

```diff
@@ -50,6 +50,7 @@ typedef enum {
 	GIT_EUNCOMMITTED    = -22,	/**< Uncommitted changes in index prevented operation */
 	GIT_EDIRECTORY      = -23,      /**< The operation is not valid for a directory */
 	GIT_EMERGECONFLICT  = -24,	/**< A merge conflict exists and cannot continue */
+  GIT_ETLSCIPHERS     = -25,  /**< No ciphers could be negotiated */
```

Comment:
Actually if we can't detect this, we should get rid of this error, maybe we'll bring it back if SecureTransport does want to play nice.